examples/lang/erc20/src/lib.rs

```diff
@@ -130,7 +128,7 @@ contract! {
         }
 
         /// Transfers token from a specified AccountId to another AccountId.
-        fn transfer_impl(&mut self, env: &mut ink_model::EnvHandler, from: AccountId, to: AccountId, value: Balance) -> bool {
```

Comment:
Why remove this path qualifier? I currently fail to see a proper import of `ink_model` to justify it.

Reply:
Okay I see that you are generating type aliases in `ink_lang`. Looks slightly intransparent to me but I don't feel like I currently understand enough to judge this decision.

Reply:
Being completely transparent would mean `ink_model::EnvHandler<SrmlEnv<NodeRuntimeTypes>>` here. A tradeoff between transparency and usability?

I suppose though that normally the user shouldn't be seeing `EnvHandler` anyway.

Reply:
> I suppose though that normally the user shouldn't be seeing EnvHandler anyway.

Yeah I would hope so but atm we cannot prevent this in all cases unfortunately.

Reply:
Type alias removed for transparency: https://github.com/paritytech/ink/pull/108/commits/48732edf535f27825aed62d31a1d0eec4eafc998